jupyter_server_proxy/__init__.py

```diff
@@ -28,8 +30,16 @@ def load_jupyter_server_extension(nbapp):
     server_handlers = make_handlers(base_url, server_proccesses)
     nbapp.web_app.add_handlers('.*', server_handlers)
 
+    # Configure SSL support
+    ssl_options = None
+    if serverproxy.https:
+        ssl_context = ssl.create_default_context(ssl.Purpose.SERVER_AUTH, cafile=serverproxy.cafile)
```

Comment:
If we setup the context like this, do we also get all the "normal" CAs or only the one in the `cafile`?

"Normal" CAs in this case would be those that your OS trusts.